model_analyzer/config/generate/perf_analyzer_config_generator.py

```diff
@@ -168,6 +189,32 @@ def set_last_results(
             self._last_results = measurement
             self._parameter_results.extend(measurement)
 
+    def _set_perf_analyzer_flags(self, model_perf_analyzer_flags: dict) -> dict:
+        # For LLM models we will be creating custom input data based on prompt length
+        perf_analyzer_flags = {k: v for k, v in model_perf_analyzer_flags.items()}
```

Comment:
Yes, since Dict is passed by reference we don't want to alter the original Dict of PA flags.

Reply:
Is it because it's used to generate multiple perf analyzer configs as in [here](https://github.com/triton-inference-server/model_analyzer/blob/d92e1d1cdda801e7ef96e1c8e776a90c29ba8dd9/model_analyzer/config/generate/model_run_config_generator.py#L104)?